joomla/joomla-cms
# [4.0] Prefill tags with most used items

Pull Request for Issue #31348 .

### Summary of Changes
Pre-fill the tags field with (default) 30 items based on the most used count.
Also reduce the min term length to 1, if there is a performance issue the site owner has to tweak this value.

### Testing Instructions
* Open an article
* Look at the tags field, you will see no tags
* Create 40 tags (or reduce the configuration entry "Initial number of shown tags" in com_tags config to a lower number
* Open an article
* You will see 30 tags
* Create several articles
* Add the always the same 10 tags
* You will always see the 10 tags and 20 additional tags
* Also test non ajax mode

### Actual result BEFORE applying this Pull Request
The tags field was empty

### Expected result AFTER applying this Pull Request
You have some (hopefully) tags to select.

### Documentation Changes Required
N/A


Thanks to @bembelimen for helping
